cmd/kubeadm/app/apis/kubeadm/bootstraptokenhelpers.go

```diff
@@ -83,7 +84,7 @@ func encodeTokenSecretData(token *BootstrapToken, now time.Time) map[string][]by
 // BootstrapTokenFromSecret returns a BootstrapToken object from the given Secret
 func BootstrapTokenFromSecret(secret *v1.Secret) (*BootstrapToken, error) {
 	// Get the Token ID field from the Secret data
-	tokenID := getSecretString(secret, bootstrapapi.BootstrapTokenIDKey)
+	tokenID := bootstrapsecretutil.GetData(secret, bootstrapapi.BootstrapTokenIDKey)
```

Comment:
all the GetData calls can be replaced with `secret.Data[...keyname...]`. If Data is nil or does not contain the specified key, that evaluates to the empty string